source/as-promise/index.ts

```diff
@@ -170,6 +170,11 @@ export default function asPromise<T>(firstRequest?: Request): CancelableRequest<
 		return promise;
 	};
 
+	promise.removeListener = (event: string, fn: (...args: any[]) => void) => {
```

Comment:
I have followed the EventEmmiter convention here in both cases. It has the `on` and the `removeListener` methods.

On one hand, it makes sense to have `on`/`off`, on the other developers will probably start by typing `remove...` as they do with `EventEmitter?

Regarding `any`/`unknown`, in my opinion, the arguments should be the same as in the `on` method. But won't be arguing much.

Please let me know what's your decision here.

Reply:
> It has the on and the removeListener methods.

Correction: `addListener` and `removeListener`

> on the other developers will probably start by typing remove... as they do with EventEmitter and browser events API?

That's a valid point, however I haven't seen anybody use the longer versions yet.

> Regarding any/unknown, in my opinion, the arguments should be the same as in the on method.

:+1:

> Please let me know what's your decision here.

Since we already have `on` and not `addListener` I'd follow the convention and go with `off`. However I don't have a strong opinion about this. /cc @sindresorhus

Reply:
> Correction: addListener and removeListener

Oh, you are right. There are all four `on`, `off`, `addListener` and `removeListener`. Then I totally agree, we already have `on` so it should be `off` 👍 Will change it.

Reply:
I agree with `off`.